client/gutenberg/editor/controller.js

```diff
@@ -331,6 +331,10 @@ export function redirectToPermalinkIfLoggedOut( context, next ) {
 	) {
 		return next();
 	}
+
+	if ( ! context.params.post ) {
```

Comment:
we check for `post` but then immediately afterwards use `context.path`. should we be checking for `path` instead?